Competition2017/src/competition/subsystems/climbing/RopeAlignerSubsystem.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.climbing;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class RopeAlignerSubsystem extends BaseSubsystem {
+    private static Logger log = Logger.getLogger(RopeAlignerSubsystem.class);
+    
+    public final XCANTalon intakeMotor;
```

Comment:
I concur with Alex's general point. Recent bots used public actuators/sensors, Earlier robots used a global map of all actuators/sensors. In all cases, we didn't get into too much trouble that I can remember.

Since your initial advocacy of this approach, when students ask how to start writing tests, I present them with two options - the public variable or the wrapper class, and talk a bit about the merits/disadvantages of both. Students naturally gravitate to the approach that makes more sense to them at the time - and thus, the one they're going to write/debug with more proficiency.